src/components/ListItem.jsx

```diff
@@ -2,7 +2,7 @@ import './ListItem.css';
 import { updateItem, deleteItem } from '../api/firebase';
 import { useState, useEffect } from 'react';
 import { getDaysBetweenDates } from '../utils/dates';
-import { Button, ListGroup, Form } from 'react-bootstrap';
+import { ListGroup, Form, Image } from 'react-bootstrap';
 
 export function ListItem({ name, itemId, dateLastPurchased, urgency }) {
```

Comment:
This may be out of scope, but should there be an easier way to add a list from `/list`, or can we only get to that feature by going through the nav menu?

<img width="622" alt="Screenshot 2023-03-01 at 10 28 53 PM" src="https://user-images.githubusercontent.com/78327387/222349019-bb749bde-bc21-4d31-80f7-0fba7078ace5.png">

Reply:
I was thinking this too while working on this issue and agree it'd feel easier to have another add item prompt inside the oven image! But like you said prior this was an issue that, in hindsight, this issue should've been broken up and I thought we could do something like this later. But I will add that as a todo stretch goal and see if I can try it out tomorrow!